### ISSUE TITLE
[JIT] ndimension not supported

### ISSUE THREAD

**Issue by eellison** [pytorch/pytorch#27505]
## 🐛 Bug

```
def fn(x)
     return x.ndimension()
torch.jit.script(fn)
```

**Comment by zou3519** [comment]
A workaround is to use `x.dim()`. `ndimension` is unofficially deprecated in favor for `dim` but I don't think anyone has actually added that deprecation warning to Python pytorch

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
